packages/toolkit/src/components/ClonePipelineDialog.tsx

```diff
@@ -38,6 +39,7 @@ import { getRawPipelineRecipeFromPipelineRecipe } from "../view";
 const ClonePipelineSchema = z
   .object({
     id: z.string(),
+    accountId: z.string(),
```

Comment:
@iamnamananand996 We call it entity instead of account. 

I think we can call it entityId under this context